docs-new/docs/graphql/core/hasura-cli/hasura.mdx

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: hasura
+sidebar_position: 3
+description: Hasura GraphQL engine command line tool using the Hasura CLI
+keywords:
+  - hasura
+  - docs
+  - CLI
+---
+
+# Hasura CLI: hasura
+
+Hasura GraphQL engine command line tool.
+
+## Synopsis
+
+    __
+     / /_   ____ _ _____ __  __ _____ ____ _
+    / __ \ / __ `// ___// / / // ___// __ `/
+    / / / // /_/ /(__  )/ /_/ // /   / /_/ /
+    /_/ /_/ \__,_//____/ \__,_//_/    \__,_/
+
+    hasura [flags]
+
+## Options
+
+    --envfile string      .env filename to load ENV vars from (default ".env")
+    -h, --help                help for hasura
+    --log-level string    log level (DEBUG, INFO, WARN, ERROR, FATAL) (default "INFO")
+    --no-color            do not colorize output (default: false)
+    --project string      directory where commands are executed (default: current dir)
+    --skip-update-check   skip automatic update check on command execution
```

Comment:
Looks like indentation is off for al

Please check in live docs to see how they look and probably wrap in (```) backticks for better readability as code blocks. Here and all other occurrences.

<img width="561" alt="Screenshot 2022-01-19 at 7 11 03 PM" src="https://user-images.githubusercontent.com/22497932/150142067-2e128f31-1359-4678-8e99-9770c407c2c7.png">

Reply:
Fixed indentation and added backticks to code blocks.